lib/JMAP/Tester.pm

```diff
@@ -245,7 +253,12 @@ sub request {
     push @suffixed, $copy;
   }
 
-  my $json = $self->json_encode(\@suffixed);
+  $request->{methodCalls} = \@suffixed;
+
+  $request = $request->{methodCalls}
+    if $ENV{JMAP_TESTER_NO_WRAPPER} && _ARRAY0($input_request);
```

Comment:
I don't understand the `_ARRAY0($input_request)` here.

I suppose what we're saying is `JMAP_TESTER_NO_WRAPPER` means "Do not wrap old-style calls" but it does *not* mean "Unwrap new style calls".

Okay.